libs/geometry/include/geometry/TangentSpaceMesh.h

```diff
@@ -313,6 +319,7 @@ class TangentSpaceMesh {
     Algorithm getAlgorithm() const noexcept;
 
 private:
+    ~TangentSpaceMesh() noexcept;
     TangentSpaceMesh() noexcept;
     TangentSpaceMesh(const TangentSpaceMesh&) = delete;
     TangentSpaceMesh& operator=(const TangentSpaceMesh&) = delete;
```

Comment:
by convention deleted methods should have `public` visibility.

Reply:
done